test/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ set(TEST_INPUTS
     strided_domain.c
     strided_domain_with_coefficients.c
     strided_domain_multi_dimensions.c
+    1mmWithOutInitStmt.c
```

Comment:
Nit: "without" is a single word in English, there is no sense in capitalizing "out"